src/components/Accordion/Accordion.jsx

```diff
@@ -0,0 +1,115 @@
+/*
+Copyright (C) 2018 The Trustees of Indiana University
+SPDX-License-Identifier: BSD-3-Clause
+*/
+import classNames from "classnames";
+import * as PropTypes from "prop-types";
+import React, { useState } from "react";
+import * as Rivet from "../util/Rivet";
+import { range } from "lodash";
+import { TestUtils } from "../util/TestUtils";
+import AccordionPanel from "./AccordionPanel";
+import AccordionPanelHeader from "./AccordionPanelHeader";
+
+const createControlId = (id, index) => `accordian_${id}_control_${index}`
+const testIds = TestUtils.Accordion
+
+/**
+ * Group content into sections that can be opened and closed
+ */
+const Accordion = ({
+  children,
+  className,
+  iconClosed,
+  iconOpened,
```

Comment:
Can we default these two per the docs sample? Thoughts?

Seems like that covers most usual cases...